DPPP/DemixWorker.cc

```diff
@@ -604,7 +604,8 @@ namespace DP3 {
       if (nrtmp == 0) {
         return 0;
       }
-      return GenSort<float>::kthLargest (tmp, nrtmp, (nrtmp-1)/2);
+      std::nth_element(tmp, tmp+(nrtmp-1)/2, tmp+nrtmp);
+      return tmp[(nrtmp-1)/2];
```

Comment:
Does this mean that `GenSort<T>::kthLargest` is missing from the new arrays implementation? This would be an API breakage to document, I'm fine with using `std::` here.

Reply:
Thanks -- GenSort is not missing from arrays2, but rather it's part of Utilities. However, the sorting routines were no longer pulled in via #include <..Arrays/Array.h>, so it gave an error. I could have fixed it with an appropriate #include, but I think it's better to use std when possible.